src/commands/showView.ts

```diff
@@ -48,7 +50,10 @@ export class ShowViewCommand extends Command {
 			case Commands.ShowAccountView:
 				return this.container.accountView.show();
 			case Commands.ShowGraphView:
-				return this.container.graphView.show();
+				if (context.type === 'scm' && context.scm !== undefined) {
```

Comment:
Let's convert the SCM object into a Repository here, and then no changes are required on the Graph

```suggestion
			case Commands.ShowGraphView: {
				let commandArgs = args;
				if (context.type === 'scm' && context.scm.rootUri != null) {
					const repo = this.container.git.getRepository(context.scm.rootUri);
					commandArgs = repo != null ? [repo, ...args] : args;
				}
				return this.container.graphView.show(undefined, ...commandArgs);
			}
```

Reply:
Our linter doesn't like declaractions in case blocks, so I'll have to move that line up, but the rest sounds good!

Reply:
Should be fine as long as it has scope, e.g. wrapped in `{}`